### PR TITLE
SYS-3338 - converted Vec params to BoundedVec params

### PR DESCRIPTION
**Pull request by ivan-cholakov**
Converted the returned signature params of transfer,lift,lower functions

### CODE REVIEW

File: pallets/token-manager/src/lib.rs [hunk 1]
```diff
@@ -596,17 +597,19 @@ impl<T: Config> Pallet<T> {
...
-        return (
-            SIGNED_TRANSFER_CONTEXT,
-            proof.relayer.clone(),
-            from,
-            to,
-            token_id,
-            amount,
-            sender_nonce,
+    ) -> BoundedVec<u8, MaximumSignedParamsSize> {
+        return BoundedVec::truncate_from(
```
Comment (thadouk) [inline pallets/token-manager/src/lib.rs:601]: Is this safe? Does the proof have a fixed size?

**Review by thadouk — COMMENTED** [review]
I think the changes need to start from a storage item cause that is where the bounds should be applied.

File: pallets/token-manager/src/lib.rs (hunk 1 above)
Comment (ivan-cholakov) [reply pallets/token-manager/src/lib.rs:601]: the tests are passing with the `pub type MaximumSignedParamsSize = ConstU32<256>;` constant

Comment (ivan-cholakov) [reply pallets/token-manager/src/lib.rs:601]: what other testing should be done here?

**Pull request closed without merging**